.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 12, 14]
+        node: [14.15.1]
```

Comment:
If only one version is tested, a matrix is not needed. But it may still be useful to test in different Node.js versions.

Also, I believe `14` would use the latest 14 version, so we don't need to set the complete version. (same below and in `.npmrc`)